data/hackathon/scenario6.mdx

```diff
@@ -25,58 +25,60 @@ Your challenge is to
+- FYI - In case you want to build your RHEL AI image yourself later, have a look at the dev preview approach here: https://github.com/RedHatOfficial/rhelai-dev-preview
 
 ## 6.1 - Setting Up
+The ACMEFS team wants to start with a true blue through-and-through open source approach. This means a model that's available as open source and which was trained on open source data and hence they want to use the Granite model familiy. For that your job as the supporting Red Hatter is to help the team show how to download a model and serve it.
 - Go to demo.redhat.com and order your teams' InstructLab RHEL VM (Nvidia/CUDA) environment
 - Use the virtual python environment if you want GPU acceleration
-- Install the instruct lab command line tooling
+- Install the instruct lab command line tooling - make sure it's version v0.16.1
 - Download the granite model from instructlab/granite-7b-lab-GGUF and name it 'granite-7b-lab-Q4_K_M.gguf'
 - Serve the Model
 
 ## 6.2 - Chat and test knowledge
+The team now wants to first try and test-drive the newly downloaded model by chatting with it to get a feel for a Granite model.
 - Chat with the model and test its knowledge about Instruct Lab 
 If you find the answers somewhat peculiar, your mission is to fix that - should you accept it. And no, this message will not self-destruct. Should you be happy with the answer you can select a different knowledge area to improve.
 
 ## 6.3 - Add new knowledge
+The real reason organisations want to use AI is because they can encode institutional knowledge that leads to either a competitive advtange or reducing cost by supporting internal processes. With finetuning this is hard to accomplish as models seem to struggle with connecting new and unknown content with existing pre-trained content. That's why model alignment is such a gamechanger. It allows organisations to encode their knowledge into an AI model, the same way RDMBS databases allowed  data or 'knowledge' to be connected and related to via foreign keys back in the days. Now you show how this is done with InstructLab by adding new knowledge to their Granite foundation model:
 - Utilise the existing InstructLab taxonomy on your image
-- Add new knowledge (the InstructLab example knowledge file is at ~/files/qna.yaml)
-- Optional, but recommended: Verify that the taxonomy tree is A-OK.
+- Add new knowledge (the InstructLab example knowledge file containing InstructLab knowledge is at ~/files/qna.yaml)
+- Show ACME how to verify that the taxonomy tree is A-OK before you proceed.
 
 ## 6.4 - Generate synthetic data
+Once you are confident that your taxonomy tree is ok you are now showing how synthetic data is generated:
 - Generate new synthetic data with a teacher model 
 - Discuss: Does synthetic data generation need a model being served? Why/Why not?
 
 ## 6.5 - Verify expected outcomes
+You now proof to the ACME FS team that there was indeed synthetic data generated:
```

Comment:
Let's think about adding some example output  / screenshots to help folks know they are on the right track :)

Reply:
added /home/instruct/instructlab/generated related information that allows teams to verify 
a) that the directory was created and
b) that there are files in there